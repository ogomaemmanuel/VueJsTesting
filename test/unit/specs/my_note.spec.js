import my_notes from '../../../src/components/notes/my-notes.vue'
import { shallow } from 'vue-test-utils'

describe('my_notes.js', () => {
    let cmp

    beforeEach(() => {
        cmp = shallow(my_notes, { // Create a shallow instance of the component
            data: {
                searchterm: 'Html5'
            }
        })
    })

    it('equals messages to ["Cat"]', () => {
        // Within cmp.vm, we can access all Vue instance methods
        expect(cmp.vm.searchterm).toEqual('Html5')
    })

    it('has the expected html structure', () => {
        expect(cmp.element).toMatchSnapshot()
    })
})