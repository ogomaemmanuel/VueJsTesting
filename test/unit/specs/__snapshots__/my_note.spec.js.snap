// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`my_notes.js has the expected html structure 1`] = `
<div>
  <!---->
</div>
`;
